migrations/versions/2c7ff9781032_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: 2c7ff9781032
+Revises: 43e8c59337ae
+Create Date: 2019-06-21 04:33:59.021835
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '2c7ff9781032'
+down_revision = '43e8c59337ae'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events', 'order_expiry_time')
+    op.drop_column('events_version', 'order_expiry_time')
+    op.add_column('settings', sa.Column('order_expiry_time', sa.Integer(), default=15, nullable=False))
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('settings', 'order_expiry_time')
+    op.add_column('events_version', sa.Column('order_expiry_time', sa.INTEGER(), autoincrement=False, nullable=True))
```

Comment:
@kushthedude one confusion. For upgrade, you chose nullable=False and for downgrade you're allowing nullable=True. Was it nullable before?

Reply:
In the before, `nullable` was not explicitly defined and it's auto-generated, So I think it would have been nullable=False

Reply:
I'll suggest you to keep the field consistent. Allowing nullable when the migration downgrade should be avoided here, IMO